db/schema.rb

```diff
@@ -116,7 +117,7 @@
     t.string   "area_unit"
   end
 
-  add_index "gardens", ["owner_id"], name: "index_gardens_on_owner_id", using: :btree
+  add_index "gardens", ["owner_id"], name: "index_gardens_on_user_id", using: :btree
```

Comment:
Err, we don't use "user" anywhere. We say "member" and the actual field here is owner_id. What is going on here?